recipes/pycryptodomex/run_test.py

```diff
@@ -0,0 +1,18 @@
+import os
+import re
+import psutil
+import platform
+
+libname = r'(libgmp.*.(dylib|so)|(mpir|gmp).*.dll)'
+
+if platform.system() == 'Darwin':
+    # ctypes.util.find_library() isn't CONDA_PREFIX aware :'(
+    # https://github.com/ContinuumIO/anaconda-issues/issues/1716
+    os.environ['DYLD_FALLBACK_LIBRARY_PATH'] = os.path.join(os.environ['CONDA_PREFIX'], 'lib')
```

Comment:
Would it be possible to patch the library to check the `CONDA_PREFIX`?

Reply:
Okay, I have added a patch now.

Reply:
Thanks for adding that. Just get a bit nervous about encouraging people to set things like library search paths outside of builds. On macOS this increasingly is a non-option thanks to SIP (e.g. [SO question]( https://stackoverflow.com/questions/39927235/alternative-for-the-dyld-library-path-trick-since-mac-os-10-11-el-capitan-with-s  )). Patch LGTM. We might benefit from raising an issue upstream to discuss better solutions.

Reply:
In the future would like us to patch `python` as Anaconda has already done so that this sort of patching is unnecessary. Raised as issue ( https://github.com/conda-forge/python-feedstock/issues/157 ).